.github/workflows/build-frequent.yaml

```diff
@@ -124,11 +124,15 @@ jobs:
       gcchash: ${{ needs.init-submodules.outputs.gcchash }}
       multilib: ${{ matrix.multilib }}
 
-  generate-issues:
+    outputs:
+      gcchash: ${{ needs.init-submodules.outputs.gcchash }}
+
+  summarize:
     if: always() # Generate github issues even when some (or all) targets fail to build
```

Comment:
Perfect! We might need more of these `if: always()` inside the workflow to ensure the summarize step _always_ runs, but we can see how it behaves and then cross that bridge when we get to it. :)

Reply:
I checked in a different branch, it seems that even adding `if: always()` inside of `summary-runner` and `build-frequent` doesn't cause any apparent error so added `if: always()` into `summary-runner` just for safety